go/service/user.go

```diff
@@ -154,7 +155,11 @@ func (h *UserHandler) LoadMySettings(ctx context.Context, sessionID int) (res ke
 	}
 	phoneNumbers, err := phonenumbers.GetPhoneNumbers(mctx)
 	if err != nil {
-		return res, err
+		if errStr := err.Error(); strings.Contains(errStr, "not enabled") {
```

Comment:
we can also check the feature flag here.

but this is ok as well

Reply:
Unable to do that because this is a flag on the server